src/views/Results.vue

```diff
@@ -366,15 +393,54 @@ export default class Results extends ResultsProps {
     // Close the dialog
     this.dialog = false;
 
-    // If just one file, focus it
-    if (ids.length === 1) {
-      this.$router.push(`/results/${ids[0]}`);
+    //enable all uploaded files
+    let filter_module = getModule(FilteredDataModule, this.$store);
+    for (let i of ids) {
+      filter_module.set_toggle_file_on(i);
+    }
```

Comment:
There's 2 spots in the code with this exact same loop, it might be worthwhile to just have the function take an array of ids you want to enable.